experiences/homepage/homepage.test.js

```diff
@@ -47,6 +47,66 @@ const homepageTest = function() {
 		expect(cardExists).equal(true);
 	});
 
+	it('when scrolling a few pixels down, the card pops up the same amount of pixels', async() => {
+		const cardExists = await isCardExisting(
+			browser,
+			config.cards.homepage.cardSelector
+		);
+
+		expect(cardExists).equal(true);
+
+		//Make the card appear a little bit
+		await scrollBy(browser, 100);
+
+		// Test scroll bounce
+		let isAtExpectedSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised
+		);
+
+		expect(isAtExpectedSnapPoint).equal(true);
+
+		await scrollBy(browser, 20);
+
+		isAtExpectedSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised - 0.1
```

Comment:
I don't like this magic number. We should calculate it as we transform percentage with `window.innerHeight`, etc. but this should be something out of the scope in my opinion.

Reply:
yep, also scrolling down 20 may not be 0.1 always, right? as the height of the screen can change depending on the devices?